pom.xml

```diff
@@ -29,6 +29,14 @@
         <resource-server.version>1.0.42</resource-server.version>
         <slf4jVersion>1.7.5</slf4jVersion>
         <spring.version>3.2.12.RELEASE</spring.version>
+        <spring-security.version>3.2.5.RELEASE</spring-security.version>
+        <dozer.version>5.5.1</dozer.version>
+        <groovy.version>2.2.2</groovy.version>
+
+        <!-- The JDBC Driver used by the portlet -->
+        <jdbc.groupId>org.hsqldb</jdbc.groupId>
```

Comment:
should this be scoped as provided so it isn't packaged in non-localhost environments?